tests/staticfiles_tests/project/documents/cached/styles.css

```diff
@@ -1 +1,19 @@
 @import url("other.css");
+
+/* @import url("non_exist.css") */
+
+/* url("non_exist.png") */
+
+@import url("other.css")
+
+/*
+
+@import url("non_exist.css")
+
+url("non_exist.png")
+
+@import url("other.css")
+
+*/
+
+@import url("other.css")
```

Comment:
Can we add a test for inline comment? e.g.
```css
@font-face {
    src: /*url("does.not.exist.E.eot")*/ url("fonts/font.eot?#iefix") format("embedded-opentype");
}
```
and also for two comments in a line, e.g.
```css
@font-face {
    src: /*comment 1*/ url("fonts/font.eot?#iefix") /*comment 2*/ format("embedded-opentype");
}
```